nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -1174,7 +1174,7 @@ struct nrf_wifi_cmd_raw_config_filter {
 	/** Wireless device operating mode filters for Promiscuous/Monitor modes. */
 	unsigned char filter;
 	/** capture length. */
-	unsigned char capture_len;
+	unsigned int capture_len;
```

Comment:
can you please map it to unsigned short to map it to upper layer mapping

Reply:
Taken care